VideoWeb/VideoWeb/Helpers/ParticipantTilePositionHelper.cs

```diff
@@ -22,7 +22,7 @@ public static string GetTiledDisplayName(ParticipantResponse participant)
             {
                 prefix = "JUDGE";
             }
-            else if (participant.HearingRole.ToLower().Trim() == "witness")
+            else if (participant.HearingRole?.ToLower().Trim() == "witness" || participant.Role == Role.QuickLinkObserver)
```

Comment:
Don't QuickLinkParticipants also behave like witnesses, should they be included here?

Reply:
Including after discussing with Rob and Lorenzo about how they behave.